activation/atxdb.go

```diff
@@ -65,6 +65,11 @@ func (db *ActivationDb) ProcessAtx(atx *types.ActivationTx) {
 	if err != nil {
 		db.log.Error("cannot store atx: %v", atx)
 	}
+
+	err = db.ids.StoreNodeIdentity(atx.NodeId)
+	if err != nil {
+		db.log.Error("cannot store node identity: %v", atx.NodeId)
```

Comment:
please include error message in print as well

Reply:
fixed